### PR TITLE
Functional tests - Add new test 'CRUD pack of products' in product page V2

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add new test 'CRUD pack of products' in product page V2
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| Related PRs       | no
| How to test?      | no need
| Possible impacts? | no

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]

**Review by Progi1984 — CHANGES_REQUESTED** [review]
@nesrineabdmouleh Need a rebase

**Review by SD1982 — DISMISSED** [review]

**Review by boubkerbribri — DISMISSED** [review]

File: tests/UI/pages/FO/product/index.ts [hunk 1]
```diff
@@ -220,6 +238,21 @@ class Product extends FOBasePage {
     };
   }
 
+  /**
+   * Get product information in pack
+   * @param page {Page} Browser tab
+   * @param productInList {object} Product in pack list
```
Comment (Progi1984) [inline tests/UI/pages/FO/product/index.ts]:
```suggestion
   * @param productInList {number} Product in pack list
```

File: tests/UI/pages/FO/product/index.ts [hunk 2]
```diff
@@ -220,6 +238,21 @@ class Product extends FOBasePage {
...
+   * @returns {Promise<{image: string, quantity: number, price: string, name: string}>}
+   */
+  async getProductInPackList(page: Page, productInList: number = 1) {
```
Comment (Progi1984) [inline tests/UI/pages/FO/product/index.ts]:
```suggestion
  async getProductInPackList(page: Page, productInList: number = 1): Promise<{image: string, quantity: number, price: string, name: string}> {
```

**Review by Progi1984 — COMMENTED** [review]
A last feedback @nesrineabdmouleh

File: tests/UI/pages/FO/product/index.ts (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/FO/product/index.ts]: Done thx :)

File: tests/UI/pages/FO/product/index.ts (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/FO/product/index.ts]: Done :)

**Review by Progi1984 — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**